### ISSUE TITLE
0.7.3 Calendar doesn't load / About-time 0.1.55

### ISSUE THREAD

**Issue by Nerull91** [DasSauerkraut/calendar-weather#149]
With the latest release of about time 0.1.55, the Calendar is stuck in a perpetual load. No console errors and using foundry 0.7.3 and PF2e system.

**Comment by sladecraven** [comment]
Yes, i confirm, latest 07.3 with 0.1.55, calendar is stuck in infinite loop.

**Comment by meditatingCybermind** [comment]
also confirmed, no errors, no idea what's happening. Calendar 3.0.4, about time 0.1.55, foundry 0.7.3, PF2E 1.10.13.2079

**Comment by sladecraven** [comment]
I was able to start the calendar (ie no more stuck) by removing the about-time and calendar-weather settings from the data/settings.db from the world (be careful if you do it).
Then the calendar is not stucked, but i am not able to save a proper time value, it resets itself to a strange value.

**Comment by mrguenther** [comment]
I have this same issue. Just in case it's relevant, I happen to be using a custom calendar.

**Comment by sladecraven** [comment]
I manage to get the Calendar/Weather back to work : 
1 - Delete all previous settings of about-time and calendar-weather in your data/settings.db of your world (take care !)
2 - Launch your world, select your calendar in about-time
3 - Load the calendar in Calendar-Weather and setup a date -> the date is not taken into account (at least on my side).
4 - Then create a macro with : 
```
Gametime.setAbsolute({
        years: 2512,
        months: 17,
        days: 30,
        hours: 10,
        minutes: 1,
        seconds: 1
      })
```
This is for a Warhammer Calendar, so of course adapt years/months/days/...  to relevant data for your world/calendar.

5 - The calendar is now displaying the proper date, and the edit function is now working also.
